src/PKSim.Presentation/DTO/DiseaseStates/DiseaseStateUpdater.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Linq;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+using PKSim.Core.Services;
+using PKSim.Presentation.DTO.Mappers;
+using PKSim.Presentation.DTO.Parameters;
+
+namespace PKSim.Presentation.DTO.DiseaseStates
+{
+   public interface IDiseaseStateUpdater
+   {
+      /// <summary>
+      ///    Updates the value in the DTO based on the origin data value (typically used in edit scenario)
+      /// </summary>
+      void UpdateDiseaseStateDTO(DiseaseStateDTO diseaseStateDTO, OriginData originData);
+
+      /// <summary>
+      ///    Updates the origin data with value from the DTO. This is typically used in save scenario
+      /// </summary>
+      void UpdateOriginDataFromDiseaseState(OriginData originData, DiseaseStateDTO diseaseStateDTO);
+
+      void UpdateDiseaseStateParameters(DiseaseStateDTO diseaseStateDTO);
+   }
+
+   public class DiseaseStateUpdater : IDiseaseStateUpdater
```

Comment:
All this logic was in the presenter for individual. I moved so that it can be reused